docs/cygnus.md

```diff
@@ -15,5 +15,5 @@ Current stable release is able to persist the following sources of data in the f
 
 ## Documentation and API
 
-FIWRE Cygnus at [Github](https://github.com/telefonicaid/fiware-cygnus).
+FIWARE Cygnus at [ReadTheDocs](https://fiware-cygnus.readthedocs.io).
 
```

Comment:
why don't have both links? repo & doc

Reply:
Fixed in 4c83c3be59504f08d85e53dd42613e65ae78e6c3